### PR TITLE
Implement more session properties

### PR DESCRIPTION
**Pull request by nokome**
Adds handling of new session properties (and corresponding `*Limit`)

- `clientsRequest`
- `durationRequest`
- `timeoutRequest`
- `networkTransferRequest`

Currently network transfer is boolean i.e. `false` when the permitted number is zero. Future work will include monitoring of transfer to make this continuous.

Some monitoring stats are affected (i.e. 'sessionsCount' now includes stopped but not 'stale' sessions). In another iteration, I'll split this count up.

Also other updates and fixes.

### CODE REVIEW

File: host/FirecrackerSession.ts [hunk 1]
```diff
@@ -102,6 +102,13 @@ export class FirecrackerSession extends Session {
     this.bootArgs = 'reboot=k panic=1 pci=off'
   }
 
+  repr(): any {
+    const { id } = this
+    return {
+      id
+    }
+  }
+
```
Comment (beneboy) [inline host/FirecrackerSession.ts]: Why not just:
`return { id: this.id }` ?

File: host/Manager.ts [hunk 2]
```diff
@@ -215,10 +313,15 @@ export class Manager extends BaseExecutor {
...
+        let { id, name } = sessionPermitted
+
         // Assign a unique, difficult to guess, identifier
         // that allows routing back to the `Session` instance
         // in the execute() method
-        const sessionId = crypto.randomBytes(32).toString('hex')
+        if (id === undefined) id = crypto.randomBytes(32).toString('hex')
+
+        // Assign a human friendly name if necessary
+        if (name === undefined) name = moniker.choose()
```
Comment (beneboy) [inline host/Manager.ts]: Docker already generates these kinds of names, could this just grab the docker name if defined?

File: host/FirecrackerSession.ts (hunk 1 above)
Comment (nokome) [reply host/FirecrackerSession.ts]: In this case, no reason. I've just been finding destructuring assignments really useful and concise in other situations (e.g. providing a default value) and was carrying on with the idiom.

File: host/Manager.ts (hunk 2 above)
Comment (nokome) [reply host/Manager.ts]: Yeah, but also wanted to have the human friendly names for Firecracker sessions

**Pull request merged**